src/server/lib/operations.py

```diff
@@ -1,5 +1,5 @@
 def addition(numbers):
-    sum = 0
+    result = 0
     for number in numbers:
-        sum += number
-    return sum
+        result += int(number)
```

Comment:
The preferred behavior would be to have this function only handle integer type data.
That would keep the sanity of the function and eventually help catch errors coming to the operation. Please revert
```suggestion
        result += number
```